examples/urca-23_starkiller/actual_network.F90

```diff
@@ -69,8 +69,7 @@ module actual_network
   integer, parameter :: i_drate_dt    = 2
   integer, parameter :: i_scor        = 3
   integer, parameter :: i_dscor_dt    = 4
-  integer, parameter :: i_dqweak      = 5
-  integer, parameter :: i_epart       = 6
+  integer, parameter :: i_eneut       = 4
```

Comment:
why are these both 4?

Reply:
I answered this below in the discussion about the template file.